tests/cpp/dlr_test.cc

```diff
@@ -69,31 +74,48 @@ TEST(DLR, TestSetDLRInput) {
   EXPECT_EQ(*(img + 224*224), *(in_img + 224*224));
   EXPECT_EQ(*(img + 224*224*3 - 1), *(in_img + 224*224*3 - 1));
   free(in_img);
+  DeleteDLRModel(&model);
 }
 
 
 TEST(DLR, TestGetDLRInputShape) {
   auto model = GetDLRModel();
-
   int64_t input_size;
   int input_dim;
   int index = 0;
   EXPECT_EQ(GetDLRInputSizeDim(&model, 0, &input_size, &input_dim), 0);
   EXPECT_EQ(input_dim, 4);
   EXPECT_EQ(input_size, 1*224*224*3);
+  int64_t shape[input_dim];
+  EXPECT_EQ(GetDLRInputShape(&model, 0, shape), 0);
+  EXPECT_EQ(shape[0], 1);
+  EXPECT_EQ(shape[1], 224);
+  EXPECT_EQ(shape[2], 224);
+  EXPECT_EQ(shape[3], 3);
+  DeleteDLRModel(&model);
 }
 
 TEST(DLR, TestGetDLROutputShape) {
   auto model = GetDLRModel();
-
   int64_t output_size;
   int output_dim;
   int index = 0;
+  // output 0
   EXPECT_EQ(GetDLROutputSizeDim(&model, 0, &output_size, &output_dim), 0);
   EXPECT_EQ(output_dim, 1);
-  
+  EXPECT_EQ(output_size, 1);
+  int64_t shape0[output_dim];
```

Comment:
Can you use `std::vector` or `new` to do this dynamic allocation? This will cause an error during windows build.

Reply:
Replaced it with `std::vector`. Btw, do we still support Mac build? I got `ld: unknown option: --exclude-libs` when building on  
- Mac OS10.14.6. 
- Apple clang version 11.0.0 (clang-1100.0.33.17).
- cmake version 3.12.1